### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by ivangonzalez224**
It is my personal portfolio project and I have added the following new features:

Added the cards needed for the works section html skeleton.
Added styles for the desktop version using media queries.
Added the folder with the images used in the desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,24 +17,24 @@
...
+                            <a href="#contact_form">Contact</a>
                         </li>
                         <li>
-                            <a class="navBtnMessage" href="#">
-                                <img class="messageBtn" src="images/mail.png" alt="">
+                            <a class="navBtnMessage" href="#contact_form">
+                                <img class="messageBtn" src="images/mail.png" alt="Contact me button">
                             </a>
                         </li>
                     </ul>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to stick to the Figma design as close as possible. Kindly look into below two screenshots and walk your design alongside the figma design.

### Your Design
your-header

### Figma Design
figma-header

File: index.html [hunk 2]
```diff
@@ -50,61 +50,64 @@
...
+                    <a href="https://www.angellist.com/">
+                        <img src="images/angelist.png" alt="Angel List Icon">
+                    </a>
+                    <a href="https://twitter.com/ivang2204">
+                        <img src="images/twitter.png" alt="Twitter Icon">
+                    </a>
+                    <a href="https://medium.com/">
+                        <img src="images/medium.png" alt="Medium Icon">
+                    </a>
                 </div>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to stick to the Figma design as close as possible. Kindly look into below two screenshots and walk your design alongside the figma design.

### Your Design
your-headline-2

### Figma Design
figma-headling

File: index.html [hunk 3]
```diff
@@ -50,61 +50,64 @@
...
-
                 <div class="headlineImage">
-                    <img src="images/illustration.png" alt="">
+                    <img src="images/illustration.png" alt="Robot Illustration">
                 </div>
-            </header>
+                <div class="headlineImageDesk">
+                    <img src="images/illustrationDesktop.png" alt="Robot Illustration">
```
Comment (Qoosim) [inline index.html]: - [ ] In your mobile version before the desktop version. You were asked to add the background image with css background-image property not with the `img`. Kindly use the same approach for your desktop version.

File: index.html [hunk 4]
```diff
@@ -135,7 +240,10 @@ <h1>About me</h1>
                     <a class="resumeBtn" href="#">Get my resume</a>
                 </div>
                 <div class="aboutImage">
-                    <img src="images/illustrationAbout.png" alt="">
+                    <img src="images/illustrationAbout.png" alt="Robot illustration">
+                </div>
+                <div class="aboutImageDesk">
+                    <img src="images/illustrationAboutDesk.png" alt="Robot illustration">
```
Comment (Qoosim) [inline index.html]: - [ ] In your mobile version before the desktop version. You were asked to add the background image with css background-image property not with the `img`. Kindly use the same approach for your desktop version.

### Your Design
your-about

### Figma Design
figma-about

- [ ] Also, it is required that your design for the desktop version must not affect your already designed mobile version. I would recommend you to check the background image for the about myself section as that has created horizontal scroll bar. Kindly attend to it and fix.

### Your App (Mobile version)
[app-behave.webm](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/26597432/16983b42-e91e-4d1b-83c4-fd51f04fcd83)

File: index.html [hunk 5]
```diff
@@ -207,8 +315,8 @@ <h1>About me</h1>
             </ul>
         </div>
     </div>
-    <form action="https://formspree.io/f/mdovrwgl" method="post">
-        <span>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</span>
+    <form action="https://formspree.io/f/mdovrwgl" method="post" id="contact_form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         <ul>
           <li>
             <input type="text" aria-label="name" name="user_name" maxlength="30" placeholder="Full name" required/>
```
Comment (Qoosim) [inline index.html:314]: - [ ] It is required to stick to the Figma design as close as possible. Kindly look into below two screenshots and walk your design alongside the figma design.

### Your Design
your-contact-form

### Figma Design
figma-contact-form

File: index.html [hunk 6]
```diff
@@ -50,61 +50,64 @@
...
         </div>
     </section>
-    <div class="works">
+    <div class="works" id="works_section">
         <div class="wrapper">
             <div class="mainWorks">
                 <h1>My Recent Works</h1>
                 <div class="workslineTop"></div>
-                <div class="worksCard">
+                <div class="worksCard worksCard_1">
```
Comment (Qoosim) [inline index.html]: - [ ] My Recent works and the horizontal line are on the same line on the Figma design while yours are on different line. Kindly fix that.
- [ ] I would like to suggest that, you reduce the margin between the line and the first project so as to match the figma design.

### Your Design
your-work-title

### Figma Design
figma-work-title

File: index.html [hunk 7]
```diff
@@ -50,61 +50,64 @@
...
+                                    <a href="https://getbootstrap.com/">Bootstrap</a>
+                                </li>
+                                <li>
+                                    <a href="https://www.ruby-lang.org/es/">Ruby</a>
+                                </li>
+                            </ul>
+                        </div>
+                        <a class="worksBtnSee" href="#">See Project</a>
+                    </div>    
+                </div>
```
Comment (Qoosim) [inline index.html:106]: - [ ] There's an horizontal scroll bar at the bottom of your page. Kindly look into that and fix.
- [ ] Also, it is required to produce a close replica of the Figma design. This includes margin, padding among others. Kindly look into the two screenshots below and walk your design alongside the figma design.

### Your Design
your-project-list

### Figma Design
figma-project-list

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Fentahunn, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -47,64 +49,63 @@
...
+                        <img src="images/linkedin.png" alt="Linkedin Icon">
+                    </a>
+                    <a href="https://www.angellist.com/">
+                        <img src="images/angelist.png" alt="Angel List Icon">
+                    </a>
+                    <a href="https://twitter.com/ivang2204">
+                        <img src="images/twitter.png" alt="Twitter Icon">
+                    </a>
+                    <a href="https://medium.com/">
+                        <img src="images/medium.png" alt="Medium Icon">
```
Comment (Shedrack-Sunday) [inline index.html:70]: - [ ] Good job making a little bit of adjustment based on the Figma design 👍 
- As you can see below, you'll agree with me that there are little adjustments let on the extra margin you have from the `nav` top to the background image of the header and where the text is placed. Kindly adjust your design while taking reference from the template provided to have a close replica 👍 

## Your work:
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @ivangonzalez224       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudos on the design and heeding to suggested changes✔️ 
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by usorfaitheloho — APPROVED** [review]
Hi @ivangonzalez224  , 
# STATUS: APPROVED! 🥇 💯 
- [x] Very Nice Design, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 9]
```diff
@@ -101,6 +102,14 @@ To run the project, do the following:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Godedict) [inline README.md:103]: dezgfhcvjbhztfi

File: style.css [hunk 10]
```diff
@@ -212,22 +308,38 @@ form span {
...
+
+  @media (min-width: 768px) {
+    display: block;
+  }
 }
 
 .navBtnMessage.active .messageBtn {
   pointer-events: none;
+
+  @media (min-width: 768px) {
```
Comment (Godedict) [inline style.css:288]: Kindly add a hover function for this card. Hover allows the card to change theme color and sticks better with the figma design.

**Review by Godedict — COMMENTED** [review]
Kindly make changes 👌

**Comment by Fentahunn** [comment]
This dosenot concern me , please donot send me again
[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
06/13/23,
06:33:12 PM

**Pull request merged**